Sources/RichTextKit/Context/RichTextContext+Actions.swift

```diff
@@ -34,6 +34,19 @@ public extension RichTextContext {
         case .stepSuperscript: return false
         case .toggleStyle: return true
         case .undoLatestChange: return canUndoLatestChange
+        case .backgroundColor: return true
```

Comment:
Perhaps this should just have a `default: true` for all actions that don't have special considerations?